src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -3770,3 +3766,17 @@ class FileSegment(BaseFileSegment):
             allow_trailing=True,
         ),
     )
+
+
+@exasol_dialect.segment()
+class EmitsGrammar(BaseFileSegment):
+    """EMITS Grammar for JSON_EXTRACT for example.
```

Comment:
I've left this with the same name but what's the difference between a"Grammar" and a "Sequence" out of interest? Should this be a Sequence?

Reply:
`Sequence` is a type of grammar (as are `AnyNumberOf`, `AnySetOf`, `Delimited`, etc..).

So calling it `EmitsGrammar` made sense when it was written like:
```python
EmitsGrammar=Sequence(
        "EMITS",
        Bracketed(Ref("UDFParameterGrammar")),
    ),
```

This is now a `Segment` (inherits from `BaseSegment`) so we should change the name to `EmitsSegment` (or something similar).